one-page.js

```diff
@@ -373,6 +365,36 @@ function inlineAssets(projectPath) {
                 }
             })();
 
+            // Add Snapchat/Google CTA code if needed
```

Comment:
Was there a reason to move it from before step 9 to after?

Reply:
Only to have the meta tags appear at the top of the file. Visually, it made more sense to me, but I'm happy to return it to its original position.